libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -95,6 +107,31 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     AP_GROUPEND
 };
 
+AC_AttitudeControl::AC_AttitudeControl( AP_AHRS &ahrs,
+                    const AP_Vehicle::MultiCopter &aparm,
+                    AP_Motors& motors,
+                    float dt) :
+#if APM_BUILD_TYPE(APM_BUILD_ArduSub)
```

Comment:
I think it may be better to create a AC_AttitudeControl_Sub class, like we have AC_AttitudeControl_Heli. It would inherit from AC_AttitudeControl (or AC_AttitudeControl_Multi if you prefer) but add new features